<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QWidget" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <widget class="QTabWidget" name="abas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>580</width>
     <height>600</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="alunos">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 161, 50);</string>
    </property>
    <attribute name="title">
     <string>Aluno</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>490</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_sobrenome">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Sobrenome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_nome">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDateEdit" name="alunos_caixa_nascimento">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);
color: rgb(102, 102, 102);</string>
          </property>
          <property name="displayFormat">
           <string>yyyy-dd-MM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="alunos_caixa_turma">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Turma</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_email">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_senha1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Senha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_senha2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Repetir Senha</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="alunos_botao_voltar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 20px;
background-color: rgb(227, 246, 0);</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="alunos_botao_cadastrar">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 20px;
background-color: rgb(11, 97, 144);</string>
          </property>
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="professores">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 161, 50);</string>
    </property>
    <attribute name="title">
     <string>Professor</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>490</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_sobrenome_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Sobrenome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_nome_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Nome</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="alunos_caixa_turma_2">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Turma</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="alunos_caixa_turma_3">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>Mat√©ria</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QListView" name="listView"/>
        </item>
        <item row="0" column="1">
         <widget class="QListView" name="listView_2"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QDateEdit" name="alunos_caixa_nascimento_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);
color: rgb(102, 102, 102);</string>
          </property>
          <property name="displayFormat">
           <string>yyyy-dd-MM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_email_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_senha1_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Senha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alunos_caixa_senha2_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;
background-color: rgb(217, 217, 217);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Repetir Senha</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="alunos_botao_voltar_2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 20px;
background-color: rgb(227, 246, 0);</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="alunos_botao_cadastrar_2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 20px;
background-color: rgb(11, 97, 144);</string>
          </property>
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
